zerver/openapi/zulip.yaml

```diff
@@ -7930,6 +7930,50 @@ paths:
                         "msg": "",
                         "result": "success",
                       }
+  /settings/drafts:
```

Comment:
I think rather than creating additional `/settings/` endpoints, we should probably just add this feature to the main `/settings` endpoint.  (the same argument holds for the existing `enter_sends`, which if you're up for it would be well worth migrating in a separate prep PR to figure out how to extend that endpoint without the complications of a new feature).  Overall it was probably a mistake to introduce separate /settings/notifications and /settings/display endpoints, but at least those are 10+ related settings each; creating a new endpoint with just 1 setting seems like it's going to clutter our API documentation for little to no benefit.

The other approach I could see taking is making a `/settings/compose` to be a bit broader than just drafts, but I'd be hard-pressed to come up with 10 more settings in that category.